components/sidebar/sidebar_category_header.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+import {DraggableProvidedDragHandleProps} from 'react-beautiful-dnd';
+
+import {wrapEmojis} from 'utils/emoji_utils';
+
+type Props = {
+    children?: React.ReactNode;
+    displayName: string;
+    dragHandleProps?: DraggableProvidedDragHandleProps;
+    isCollapsed?: boolean;
+    isCollapsible?: boolean;
+    isDragging?: boolean;
+    isDraggingOver?: boolean;
+    onClick?: (event: React.MouseEvent<HTMLElement>) => void;
+}
+
+const SidebarCategoryHeader = React.forwardRef((props: Props, ref?: React.Ref<HTMLDivElement>) => {
```

Comment:
I pulled this out of the `SidebarCategory` into a reusable component so that I didn't need to repeat the styles in the `UnreadChannels` component